homeassistant/components/matrix/manifest.json

```diff
@@ -2,8 +2,7 @@
   "domain": "matrix",
   "name": "Matrix",
   "documentation": "https://www.home-assistant.io/integrations/matrix",
-  "requirements": ["matrix-client==0.4.0"],
+  "requirements": ["matrix-nio==0.19.0", "pillow==9.3.0"],
```

Comment:
Why add pillow ?

Reply:
Specific reasoning: to adhere as much as possible to `matrix-nio`'s example code of how to upload an image (as now implemented in `matrix.__init__.MatrixBot._send_image`). Certain image metadata (namely height/width) is required to be manually constructed as part of using this library, and it is highly undesirable to omit it when sending images over matrix.
https://matrix-nio.readthedocs.io/en/latest/examples.html#sending-an-image

Generic reasoning: I didn't think it would be good practice to attempt to read image attributes (namely height/width) in any other way than using a dedicated library like `pillow`. If there is a suggested way to achieve this more generically to homeassistant that would allow this dependency to be dropped, I'm happy to make that change